client/influxdb.go

```diff
@@ -338,6 +359,10 @@ func (p *Point) MarshalJSON() ([]byte, error) {
 	return json.Marshal(&point)
 }
 
+func (p *Point) MarshalString() string {
+	return tsdb.NewPoint(p.Name, p.Tags, p.Fields, p.Time).String()
```

Comment:
So when we create our "public" binary protocol we'll probably switch to using that endpoint so we don't have so much running around.  Lot of extra work here when we already have a concrete type at the client library.  Nothing to do now of course.